pandas/_libs/lib.pyx

```diff
@@ -119,12 +124,42 @@ def is_scalar(val: object) -> bint:
     - instances of decimal.Decimal
     - Interval
     - DateOffset
+    - Fraction
+    - Number
 
+    Returns
+    -------
+    True if the given value is scalar, False otherwise.
```

Comment:
In the returns first list the type (i.e. `bool`) in the next line indented the description.